content/en/serverless/step_functions/installation.md

```diff
@@ -66,6 +63,127 @@ Then, to send your Step Functions logs to Datadog:
 
    <div class="alert alert-info">If you enable tracing (which automatically includes enhanced metrics), you are billed for both Serverless Workload Monitoring and Serverless APM. See <a href="https://www.datadoghq.com/pricing/?product=serverless-monitoring#products">Pricing</a>.</div>
 
+[6]: /logs/guide/forwarder
+[11]: /serverless/step_functions/merge-step-functions-lambda
+[16]: /logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination
+[1]: /getting_started/integrations/aws/
+[2]: https://app.datadoghq.com/integrations/aws
+{{% /tab %}}
+{{% tab "Datadog CI" %}}
+1. If you have not already, install the [Datadog CLI][1] v2.18.0+.
+
+   ```shell
+   npm install -g @datadog/datadog-ci
+   ```
+
+1. Ensure you have deployed the [Datadog Lambda Forwarder][2], and that you are using v3.130.0+. You might need to [update your Forwarder][2]. As an alternative, you may also use [Amazon Data Firehose][3], which can subscribe to Amazon CloudWatch log groups across multiple AWS regions. Your Step Functions log group name should use this format: 
+   ```
+   /aws/vendedlogs/states/<STEP_FUNCTION_NAME>-Logs
+   ```
+
+   **Optionally**, you can add your environment to this log group name: `/aws/vendedlogs/states/<STEP_FUNCTION_NAME>-Logs-<ENV>`
+
+   Take note of your Forwarder's ARN.
+
+1. Add the following to your `serverless.yml`:
+
+   ```shell
+   datadog-ci stepfunctions instrument \
+    --step-function <STEP_FUNCTION_ARN> \
+    --forwarder <FORWARDER_ARN> \
+    --env <ENVIRONMENT> \
+    --propagate-upstream-trace \
+    --merge-step-function-and-lambda-traces
+   ```
+
+   - Replace `<STEP_FUNCTION_ARN>` with the ARN of your Step Function. Repeat the `--step-function` flag for each Step Function you wish to instrument.
+    - Replace `<FORWARDER_ARN>` with the ARN of your Datadog Lambda Forwarder. This step configures the log stream subscription for the Forwarder.
+   - Replace `<ENVIRONMENT>` with the environment tag you would like to apply to your Step Functions.
+
+    For more information about the `datadog-ci stepfunctions` command, see the [Datadog CLI documentation][4].
+
+1. Set up tags. Open your AWS console and go to your Step Functions state machine. Open the *Tags* section and add the following key-value pairs:
+   | key | value |
+   | --  | ----- |
+   | `env` | Name of your environment. Required to see traces in Datadog. Defaults to `dev`. |
+   | `service` | Name of your service. Defaults to the state machine's name. |
+   | `dd_version` | `dd_sls_plugin`| 
+
+1. To enable tracing, you have two options:
+   - **Per Step Function**: Add the `DD_TRACE_ENABLED` tag to each Step Function and set the value to `true`.
```

Comment:
We can add something like: The `datadog-ci stepfunctions instrument` command already does this, so no extra work is needed.